.github/workflows/watch-conda-forge.yml

```diff
@@ -0,0 +1,104 @@
+# Check for new versions of dask on Conda Forge
+name: WatchCondaForge
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    if: github.repository == 'conda-forge/pangeo-dask-feedstock'
```

Comment:
i guess this is to prevent a bunch of PRs in forks, but you could comment for now to test in your fork, yes?